### ISSUE TITLE
Renaming Project to Rhyme only

### ISSUE THREAD

**Issue by Suzie97** [RhymePlayer/Rhyme-Player#13]
The modern day trend is that most of the projects don't really include a description of the app in the name. For example, Spotify is not called Spotify Music Player, Chrome is not called Chrome Web Browser, Akira is not called Akira Design (or something).

I also think most people will refer to the app as Rhyme and not Rhyme Player because it is simpler and more convenient.

The user will know what the apps is about from its description. Also, the name Rhyme does indicate that it is something related to music.

**Comment by Abdallah-Moh** [comment]
Naming It `Rhyme` will be hard to tell if it is a music player

**Comment by Abdallah-Moh** [comment]
@Suzie97 Do you have another name

**Comment by bodamat** [comment]
I agree. When I saw the name Rhyme (without "player". I'm lazy to read :D) first what comes to my mind is a music player. Also in some way similar to the name Rhythmbox

**Comment by Suzie97** [comment]
> Naming It `Rhyme` will be hard to tell if it is a music player

Umm, I'm not sure. @bodamat has stated that the name Rhyme made him think that the app is a music player. But we need more opinions. Let's see.
